js-packages/@prestojs/ui/src/formatters/TimeFormatter.tsx

```diff
@@ -34,11 +34,11 @@ export default function TimeFormatter(props: TimeFormatterProps): string | null
     if (!value) return null;
 
     // we use a dummy date here as the day's irrelevant for toLocaleTimeString
-    if (Number.isNaN(Date.parse(`0999-04-11T${value}`))) {
+    if (Number.isNaN(Date.parse(`2020-04-11T${value}`))) {
```

Comment:
Perhaps use epoch date instead? `1970-01-01`

Reply:
works for me, done